Makefile

```diff
@@ -66,6 +66,6 @@ test-bin:
 	@echo "Making test binary"
 	hack/build-test-bin.sh
 
-image-local:
-	@echo "Making image local"
-	$(IMAGE_BUILD_CMD) build --no-cache -f Dockerfile -t cnf-tests-local . 
\ No newline at end of file
+cnf-tests-local:
+	@echo "Making cnf-tests local"
+	$(IMAGE_BUILD_CMD) build --no-cache -f cnf-tests/Dockerfile -t cnf-tests-local . 
```

Comment:
should we have a variable for the Dockerfile name?

Reply:
Not sure about that. This is intended to build it locally, and I expect it to use this dockerfile. The other one is going to be used by CI to build the "openshift" version